integration-test/473-landuse-tier.py

```diff
@@ -13,8 +13,7 @@
 # area 30089300
 assert_has_feature(
     8, 56, 115, 'landuse',
-    { 'kind': 'national_park', 'id': -921675, 'tier': 1,
-      'min_zoom': 8 })
+    { 'kind': 'national_park', 'tier': 1, 'min_zoom': 8 })
 assert_no_matching_feature(
     7, 28, 57, 'landuse',
     { 'kind': 'national_park', 'id': -921675 })
```

Comment:
I think I see what you mean here now. Unfortunately, when I take it out the test fails, because it picks up a different `national_park`.

```
Found feature matching properties {'kind': 'national_park'} in layer 'landuse', but was supposed to find none. For example: {u'kind': u'national_park', u'area': 262412992, u'sort_rank': 18, u'source': u'openstreetmap.org', u'min_zoom': 5.0, u'tier': 1, u'id': -930900, u'osm_relation': True}
```